apps/angular/ngfor-enhancement/src/app/common/directives/ngForEmpty.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { NgForEmptyDirective } from './ngForEmpty.directive';
+
+describe('NgForEmptyDirective', () => {
+  it('should create an instance', () => {
+    const directive = new NgForEmptyDirective();
+    expect(directive).toBeTruthy();
+  });
+});
```

Comment:
When you don't really test your code, it's better to delete useless files.

Reply:
i noticed this as well, i will remove unnecessory test files,